### ISSUE TITLE
FLEETONE : Ajout de marques dans la table

### ISSUE THREAD

**Issue by PIEGAY** [groupe-gac/FleetOne#129]
Nicolas, 

Est-ce que tu peux ajouter les marques suivantes : 
- SVETRUCK
- JUNGHEINRICH
- 2MB INTERNATIONALE
- STACKER
- CHARLATTE

Merci d'avance

**Comment by Portekoi** [comment]
Comment ajouter une marque?  Il faut aussi charger les véhicules/modèles depuis une source extérieure pour les ajouter dans `MakeVINLink`? Depuis Tecdoc par exemple?

**Comment by MamboJoel** [comment]
Non, c'est simplement des insert dans la table `MakeVinLink`, il faut aussi renseigner le champ `VINPrefix`. Lorsqu'on ne l'a pas, on met simplement les trois premiers caractères du nom de la marque.
